### DISCUSSION TITLE
GitHub Action - Metrics

### DISCUSSION THREAD

**Discussion by HackerSM9** [lowlighter/metrics#1321] [Q&A]
How & Where I can get Metrics Token?

**Comment by lowlighter** [comment]
Hi 👋 !

You can follow [`1️ Create a GitHub personal token`](https://github.com/lowlighter/metrics/blob/master/.github/readme/partials/documentation/setup/action.md#1%EF%B8%8F-create-a-github-personal-token) documentation to generate a token that can be used by *metrics*.

Note that the fine-grained tokens are not supported yet, so you'll need to create a classic one when prompted:
![image](https://user-images.githubusercontent.com/22963968/207932459-2720c5e3-62af-4e4b-ba58-fea6ba9301d5.png)

Hope it helps 😄 !

**Comment by HackerSM9** [comment]
Thank you so much for your response

Regards,
SM9
________________________________
